UnitTestFrameworkPkg/ReadMe.md

```diff
@@ -1611,7 +1611,7 @@ Non-Host-Based (PEI/DXE/SMM/Shell) Tests for a Functionality or Feature   | Simi
 
       GeneralPurposeLib/              # Host-Based Test for any implementation of GeneralPurposeLib
         GeneralPurposeLibTest.c
-        GeneralPurposeLibHostUnitTest.inf
+        GeneralPurposeLibHostUnitTest

```

Comment:
We would seem to want to keep this, too?

Reply:
Fixed.